_data/icons/orl.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmNsuuBBTHErnuFDcdyzaY8CKoVJtobsLJx2WQjaPjcp7g",
```

Comment:
where did you pin this CID? I cannot resolve it

Reply:
I made a pin on infura: https://orlando.infura-ipfs.io/ipfs/QmNsuuBBTHErnuFDcdyzaY8CKoVJtobsLJx2WQjaPjcp7g

Reply:
Please check it.